src/node_mgmt/env_scan.py

```diff
@@ -379,7 +386,9 @@ def do_scan(self):
                 'hosts': network_hosts
             }],
             'serial_scan':
-            serial_devices
+            serial_devices,
+            'speedwire_serials':
```

Comment:
This makes sense. Let's just fix the formatting of this block. (i.e. `'key': value,` on each line